packages/rxstats/package.json

```diff
@@ -35,7 +35,7 @@
     "rxjs": "^6.5.3"
   },
   "scripts": {
-    "build": "rm -rf ./build/* && babel ./src -d ./build --ignore *.test.js",
+    "build": "BABEL_ENV=build babel src --out-dir build --source-maps --extensions .js --config-file ../../babel.config.js --delete-dir-on-start --no-comments",
     "lint": "eslint -c ../../.eslintrc ./src",
     "test": "mocha --exit ./src/**/*.test.js ./src/*.test.js",
     "test:watch": "yarn test -- --watch",
```

Comment:
Lerna might have a way to make this build command consistent for all of the packages... but for now, the best way I know of to do it is just to define it in every package...